docs/en/enterprise-edition/rn/look-ahead-planned-updates-prisma-cloud/look-ahead-secure-the-runtime.adoc

```diff
@@ -99,55 +81,29 @@ The ‘purl’ field identifies the absolute path for the packages.
-
-//CWP-56591 [Doc Ticket] CWP-50970 [Eng Ticket]
-//|*Inconsistency with jar packages names*
-//|Fixed an issue where old jar versions were not parsed properly.
+//CWP-56591
+//TODO:Pending blurb approval
+|*Enhanced CVE Detection for Older JAR Versions*
+|Fixed an issue where incorrect parsing of older JAR file versions resulted in missing CVE entries.
 
-//CWP-55859 [Eng Ticket] CWP-56294 [Doc Issue]
-//FIXME: No Doc ticket
-|*`incidentTime` (timestamp) macro for Webhooks*
-|The `incidentTime` macro is available for webhook alerts. The macro shows the time the incident occurred. For example, `Jan 21, 2018 UTC`.
+//CWP-58016
+|*Add ‘risk factor’ to the CSV downloads for Functions and host*
+|Fixed an issue where “risk factor” field was missed on Function and Host CSV results.
 
-//CWP-56589[Doc Ticket] CWP-49010 [Eng Ticket]
-//TODO: Pending final approval post edit.
-|*Twistcli Image Scanning*
-|Resolved an issue where container images scanned with twistcli were not appearing in the Runtime Security, *Monitor >Vulnerabilities >Images >CI*. Use  `--build` and `--job` flags to include build number and job name information on *Runtime Security > Inventory*.
-
-//CWP-56554 [Doc Ticket] CWP-54108 [Eng Ticket]
-//TODO: Pending approval
-|*Fixed Containerized Scan Failure*
-|Resolved issue causing containerized scans to fail due to long JSON, particularly when encountering large Java dependency lists. You can now conduct scans without encountering this issue.
-
-//CWP-56787[Doc Ticket] CWP-56697 [Eng Ticket]
-//TODO: Pending approval
-|*Support to download Serverless Defender Bundle*
-|Prisma Cloud console supports downloading the serverless defender bundle for C# on Azure and AWS. This enhancement resolves a prior runtime error that prevented users from downloading the bundle.
-
-//CWP-56786 [Doc Ticket]
-//TODO: Pending Blurb
-
-//CWP-56784 [Doc Ticket] CWP-54107 [Eng Ticket]
+//CWP-56784
 //TODO: Pending Approval
-//|*Improved Handling of Rejected CVEs*
-//|With the transition to the CVE 5.0 dataset, NVD has updated the format of rejected CVE descriptions. Prisma Cloud now seamlessly identifies 'Rejected' and 'Disputed' statuses of CVEs. The 'Rejected' status is now labeled as 'Rejected reason', while 'Disputed' status is indicated as a tag, ensuring accurate vulnerability assessment.
+|*Improved Handling of Rejected and Disputed CVEs*
+|With the transition to the CVE 5.0 dataset, NVD has updated the format of rejected CVE descriptions.
+Prisma Cloud now seamlessly identifies 'Rejected' and 'Disputed' statuses of CVEs. The 'Rejected' status is now labeled as 'Rejected reason' in NVD, while CVEs tagged as 'Disputed' are identified using cve.org data source, ensuring accurate vulnerability assessment.
```

Comment:
Suggested edit:

The 'Rejected' status is **now** (repetitive IMO) labeled as 'Rejected reason' in NVD

Reply:
@jenjoe22 - Addressed in [commit](https://github.com/hlxsites/prisma-cloud-docs/pull/572/commits/aaea3e26c9ed4f4d5dd35de0569c20de44d5820d).